lib/util/SortableSet.js

```diff
@@ -50,16 +75,14 @@ class SortableSet extends Set {
 		this._invalidateCache();
 	}
 
-	/**
-	 * @returns {void}
-	 */
 	sort() {
 		this.sortWith(this._sortFn);
 	}
 
 	/**
-	 * @param {Function} fn - function to calculate value
-	 * @returns {TODO} - returns result of fn(this), cached until set changes
+	 * Get data from cache
+	 * @param {function(SortableSet<T>): T[]} fn function to calculate value
+	 * @returns {TODO} returns result of fn(this), cached until set changes
```

Comment:
I think you can return T from here and `getFromUnorderCache`, right?

Reply:
Thanks. I think `T[]` is more accurate. It blow up in some other place. Fun to see code inconsistencies when you add types!